WalletWasabi.Gui/Controls/WalletExplorer/HistoryTabViewModel.cs

```diff
@@ -121,7 +121,7 @@ private async Task TryRewriteTableAsync()
 			}
 			catch (Exception ex)
 			{
-				Logger.LogError<HistoryTabViewModel>($"Error while RewriteTable on HistoryTab: {ex}.");
+				Logger.LogError($"Error while {nameof(TryRewriteTableAsync)}: {ex}.");
```

Comment:
Where the error happened can be seen in the exception stack trace. Also I think it is better to pass the exception as an object instead of string because in that way the Logger could decide how to display it better. Please consider changing it by something like the following:

```suggestion
				Logger.LogError(ex);
```